docs/intro/common-tricks.md

```diff
@@ -144,61 +142,62 @@ f(i, 0, a.size()) { ... }
 #endif
 ```
 
- `#ifdef` 会检查程序中是否有通过 `#define` 定义的对应标识符，如果有定义，就会执行下面的内容， `#ifndef` 恰恰相反，会在没有定义相应标识符的情况下执行后面的语句。
+`#ifdef` will check whether there is a corresponding identifier defined by `#define` in the program, if there is, it will execute the following content, `#ifndef`, on the other hand, will be executed without defining the corresponding identifier The following statement.
```

Comment:
```suggestion
`#ifdef` will check whether there is a corresponding identifier defined by `#define` in the program, if there is, it will keep the following content, `#ifndef`, on the other hand, will be kept without defining the corresponding identifier The following statement.
```

原文有无，这里用 执行 不太恰当，应该用 保留